core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -1009,20 +1008,13 @@ public void testSingleValueFieldWithExtendedBoundsTimezone() throws Exception {
 
         DateMathParser parser = new DateMathParser(Joda.getStrictStandardDateFormatter());
 
-        final Callable<Long> callable = new Callable<Long>() {
```

Comment:
I'm not sure how I can inject the fake now into in QueryShardContext that is created for the request as it would mean getting hold of and changing the IndexService or ThreadPool for each node. So I think we need `System::currentTimeMillis` here.